src/vue-components/popup.vue

```diff
@@ -175,6 +190,10 @@
       morphDataReady: function () {
         return this.data.morphDataReady
       },
+      noLanguage: function () {
+        console.log('**********noLanguage', this.data.currentLanguage)
+        return this.data.currentLanguage === "undefined" || this.data.currentLanguage === undefined
```

Comment:
Does the first condition checks for `currentLanguage` having a string value of "undefined"? If so, it can be confusing, we probably better use a regular `undefined`. Or should the first condition be `typeof this.data.currentLanguage === 'undefined'`?